### PR TITLE
Fix issue regarding lost mount path

### PR DESCRIPTION
**Pull request by dplewis**
Can be tested by setting process.env.MOUNT_PATH.

Test Added

### CODE REVIEW

**Comment by dplewis** [comment]
@acinader How does this look? That TODO is a reminder for future e2e tests.

I’d recommend giving it a shot when you have time. I ran into a few issues when I tried to mount a parse server.

**Comment by acinader** [comment]
@dplewis I've taken a first (second and third) pass at testing and I have run into problems.  I don't know if it is because of this change combined with how I run the dashboard or if it is my unfamiliarity with how to test the dashboard.  I suspect the later.

Here's my setup:

I have a repo with package.json that includes the dependency:  `parse-dashboard: latest`.  I have a `parse-dashboard-config.json` file that has entries for all my pars servers.

The package.json has this script:

`"start": "parse-dashboard --config parse-dashboard-config.json",`

When I try to change the dependency from `latest` to `github:parse-community/parse-dashboard#mount-fix`

I get a blank white page with following js error in the console: `Uncaught SyntaxError: Unexpected token <`

if I click through to login.bundle.js in the inspector, they look very different, so I suspect the issue is that I don't know how to test a branch and build it, etc.  

I'll give it another try in a few hours when I have some time to look into it.

**Comment by dplewis** [comment]
@acinader There is a known issue with that https://github.com/parse-community/parse-dashboard/issues/1049 I fixed that in my last commit try to install from branch again.

**Comment by acinader** [comment]
ahh, much better!  thanks.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ npm-debug.log
 *.swp
 .env
 .idea/
+
+logs/
+test_logs
```
Comment (acinader) [inline .gitignore]: add trailing cr

**Comment by acinader** [comment]
ps: thanks for spoon feeding me.  much appreciated...

**Comment by larcoamir** [comment]
> @acinader There is a known issue with that #1049 I fixed that in my last commit try to install from branch again.

Could please you suggest me git commands to install from the branch?

**Comment by mtrezza** [comment]
@larcoamir I've noticed you've left several comments on years old threads. Your chances to receive support this way are rather slim as we use GitHub only for code related issues, not for support. For help with Parse Platform we recommend our [community forum](https://community.parseplatform.org).

**Pull request merged**